cloudgrep/tests/test_unit.py

```diff
@@ -20,20 +23,20 @@ def test_true(self) -> None:
         self.assertEqual(1, 1)
 
     def test_weird_files(self) -> None:
-        for filename in os.listdir("./tests/data/"):
+        for filename in os.listdir(f"{BASE_PATH}/data/"):
```

Comment:
This is due I've changed the path for tests, and this makes universal, so in case you want to refactor the project, it will assume a relative path.